src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,241 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.scene.shape.Circle;
+import javafx.stage.Stage;
+
+/**
+ * Main entry point of the bot.
+ */
+public class Duke extends Application {
+    private final Ui ui = new Ui();
```

Comment:
Should "ui" be changed to "UI" since constant names should be all uppercase (according to coding standard)?